### PR TITLE
add npm autoupdate and homepage info for backbone.stickit

### LINKED ISSUE #7731: [Request] update backbone.stickit to v0.9.2
**Library name:** backbone.stickit 
**Git repository url:** https://github.com/NYTimes/backbone.stickit
**npm package url(optional):** https://www.npmjs.com/package/backbone.stickit
**License(s):** MIT
**Official homepage:** https://nytimes.github.io/backbone.stickit/
**Wanna say something? Leave message here: Thanks!**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dannyxx001**
PR for #7731 
@pvnr0082t please help me review this PR, thanks.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/NYTimes/backbone.stickit
 * Official website (optional, not the repository): https://nytimes.github.io/backbone.stickit/
 * NPM package url (optional): https://www.npmjs.com/package/backbone.stickit
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 1647
   - Count of watchers: 83
   - Count of forks: 188
 * NPM download stats (optional):
   - Downloads in the last day: 139
   - Downloads in the last week: 1273
   - Downloads in the last month: 6099

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [ ] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [x] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #7731, cc @delambo

### CODE REVIEW

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/backbone.stickit/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "backbone.stickit",
-  "description": "Model binding in Backbone style.",
   "filename": "backbone.stickit.min.js",
+  "description": "Model binding in Backbone style.",
```
Comment (PeterDaveHello) [inline ajax/libs/backbone.stickit/package.json]: Any reason to touch `description` here?

Comment (dannyxx001) [inline ajax/libs/backbone.stickit/package.json]: Uh... I just want to make name and filename be the first and second line, so if it's unnecessary, I can modify it back.

Comment (PeterDaveHello) [inline ajax/libs/backbone.stickit/package.json]: Let's make it back.

Reasons:
1. It's not necessary.
2. You don't even mention this change in the commit message.

Comment (dannyxx001) [inline ajax/libs/backbone.stickit/package.json]: OK, I had turned it back.

File: ajax/libs/backbone.stickit/package.json [hunk 2]
```diff
@@ -14,5 +14,15 @@
     "type": "git",
     "url": "git://github.com/nytimes/backbone.stickit"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "homepage": "https://nytimes.github.io/backbone.stickit/",
```
Comment (PeterDaveHello) [inline ajax/libs/backbone.stickit/package.json]: Same problem here, you don't even mention this change.

**Pull request merged**